### ISSUE TITLE
SPTDataLab_libraries Section of the Repository

### ISSUE THREAD

**Issue by hfduarte** [most-ieeta/SPT-DataLab#18]
Some dlls in the SPTDataLab_libraries Section of the Repository  are no longer needed!

Potentially not needed/used:

Microsoft.VisualStudio.TestTools.CppUnitTestFramework.dll (not used anymore)
jli.dll                              (what is this? Is this for SPTMesh?)
jni_dll.dl                        (I think that this is an old jni test prior to java_mesh.dll)
mesquite.dll                 (not used anymore)
msvcr100.dll                 (what is this? Is this for SPTMesh?)
verdict.dll                     (not used anymore)

**Comment by bsilva3** [comment]
Please, remove all Dll's you consider to be unnecessary for the SPTMesh library :)
All Dll's were scattered in folders and i did my best to assemble them in one place, it is not any surprise if there are unnecessary DLL's

**Comment by hfduarte** [comment]
I realized that the app is using an old version of sptmesh. So some dll are still needed.

Microsoft.VisualStudio.TestTools.CppUnitTestFramework.dll (needed)

jli.dll (not needed)

jni_dll.dl (not needed)

mesquite.dll (needed)

msvcr100.dll (This is the Microsoft Visual C++ Redistributable, I think. This should probably be installed on the system and not distributed with the app source code)

verdict.dll (needed)

Tested on my system with the code and dlls from the repo.
